lib/generator/exampleRule.ts

```diff
@@ -1,7 +1,7 @@
 import { CacheItem } from "./exampleCache"
 export interface ExampleRule  {
```

Comment:
Why not use type and make exampleNamePostfix as optional field?

Reply:
The two filed are required,  using interface is considering that the rule may be extended in the future